### DISCUSSION TITLE
Using seaweedfs as Nextcloud backend

### DISCUSSION THREAD

**Discussion by pio2398** [seaweedfs/seaweedfs#2100] [Q&A]
I am trying to use seaweedfs as a storage for Nextcloud. I started seaweedfs with command:
```/weed server -dir=data  -s3 -s3.config=auth_s3.json -master.volumeSizeLimitMB=100``` and added external S3 storage in Nextcloud.
Connection seem to work because I can upload file but I can't even create a directory. Remove operation is also pretty unstable.
Does anyone have experience with using NextCloud with seaweedfs?

**Comment by riccardocapra-bs** [comment]
Same problem here.
